services/app/src/components/gameFooter/GameFooter.jsx

```diff
@@ -18,9 +20,11 @@ function GameFooter({ skipCurrentPlayerMove }) {
         <button type="button" className="finish-btn" disabled={!lastTurn || isAllTurnsMade || (finished && !isAllTurnsMade)} onClick={finishMove}>
           Finish Move
         </button>
-        <button type="button" className="skip-btn" disabled={finished} onClick={() => skipCurrentPlayerMove()}>
-          Skip Move
-        </button>
+        <Tooltip title="Skips your move. The game continues" className="tooltipClass" overlayInnerStyle={{fontSize: 15}} placement="rightTop" mouseEnterDelay = "0.001" mouseLeaveDelay= "0.001">
```

Comment:
I'd suggest to stick to the default enter and leave delays. 

Also, if you think we'll need to reuse some props in different places, we could move this Tooltip component into a wrapper Tooltip component that would pass these common props as defaults.

Apart from that, could we manage font size with the `tooltipClass` instead of passing inline styles?